src/views/AddTimerHomeView.tsx

```diff
@@ -0,0 +1,422 @@
+              <option value="xy">XY Timer</option>
+              <option value="tabata">Tabata</option>
+            </select>
+          </label>
+        </InputGroup>
+
+        {type === "countdown" && (
+          <InputGroup>
+            <label>
+              Duration (seconds):
+              <input
+                type="number"
+                value={duration}
+                onChange={(e) => setDuration(Number(e.target.value))}
+                min={1}
+              />
+            </label>
+          </InputGroup>
+        )}
+
+        {type === "xy" && (
+          <>
+            <InputGroup>
+              <label>
+                Round Duration (seconds):
+                <input
+                  type="number"
+                  value={duration}
+                  onChange={(e) => setDuration(Number(e.target.value))}
+                  min={1}
+                />
+              </label>
+              <label>
+                Number of Rounds:
+                <input
+                  type="number"
+                  value={rounds}
+                  onChange={(e) => setRounds(Number(e.target.value))}
+                  min={1}
+                />
+              </label>
+            </InputGroup>
+          </>
+        )}
+
+        {type === "tabata" && (
+          <>
+            <InputGroup>
+              <label>
+                Work Time (seconds):
+                <input
+                  type="number"
+                  value={workTime}
+                  onChange={(e) => setWorkTime(Number(e.target.value))}
+                  min={1}
+                />
+              </label>
+              <label>
+                Rest Time (seconds):
+                <input
+                  type="number"
+                  value={restTime}
+                  onChange={(e) => setRestTime(Number(e.target.value))}
+                  min={1}
+                />
+              </label>
+            </InputGroup>
+            <InputGroup>
+              <label>
+                Rounds:
+                <input
+                  type="number"
+                  value={rounds}
+                  onChange={(e) => setRounds(Number(e.target.value))}
+                  min={1}
+                />
+              </label>
+            </InputGroup>
+          </>
+        )}
+
+        <AddButton onClick={handleAddTimer}>Add Timer</AddButton>
+      </ConfigSection>
+
+      <TimerQueue>
+        <h2>Workout Queue ({state.timers.length} timers)</h2>
+        <div>
+          <ControlButton className="start" onClick={startWorkout}>
+            {isWorkoutRunning ? 'Pause' : 'Start Workout'}
+          </ControlButton>
+          <ControlButton className="reset" onClick={resetWorkout}>
+            Reset
+          </ControlButton>
+          <ControlButton className="forward" onClick={fastForward}>
+            Skip Timer
+          </ControlButton>
+        </div>
+
+        {state.timers.map((timer, index) => (
```

Comment:
A display of `total work time` is missing

Reply:
🤦🏾‍♂️ I hate when I forget things. I'm terrible at keeping tabs. I'm trying to get better at using notion to make a bullet list or use the feature in GitHub to make a checklist

Reply:
Always a challenge!